pkg/v0/cmd.go

```diff
@@ -332,7 +336,7 @@ func cherryPick(ctx context.Context, logger *logrus.Entry, c internal.Commit, co
 		}
 	}
 
-	for _, cmd := range []*exec.Cmd{
+	gomod := []*exec.Cmd{
```

Comment:
I am surprised that you only want to delay generating manifests in the root - why doesn't this apply equally to the first three commands in this list - which run in the root, not the staging repos?

Reply:
After additional testing, it was determined that only the "generate manifests" needed to be delayed.

Generate manifests actually does a build, and thus needs things all sync'd properly (in this case all the commits).

In the cases of `go mod`, we still want those updated files associated with the original commit, to localize the changes, and as you had pointed out earlier, to possibly keep revert as a possibility.

Reply:
Well, we lose the ability to revert either way with this PR. If you think just delaying the build until the end will make the automated pick work better, then let's do that. Snipping those deps to the root will be ideal as a longer-term solution.

Reply:
That being said, there may still be a need in the future, to enhance this with a `-delay-go-mod`-type-of-option, but not now.